assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -410,6 +515,16 @@ def save_as_mlflow(self):
         self._hf_config_cls = self._hf_config_cls if self._hf_config_cls else AutoConfig
         self._hf_tokenizer_cls = self._hf_tokenizer_cls if self._hf_tokenizer_cls else AutoTokenizer
 
+        if self._task == SupportedNLPTasks.TEXT_GENERATION.value:
+            inputs = Schema([ColSpec(DataType.string)])
+            outputs = Schema([ColSpec(DataType.string)])
+            params = ParamSchema([ParamSpec("top_p", "float", default=1.0),
+                                  ParamSpec("temperature", "float", default=1.0),
+                                  ParamSpec("max_new_tokens", "integer", default=50),
+                                  ParamSpec("do_sample", "boolean", default=True),
+                                  ParamSpec("return_full_text", "boolean", default=True)])
+            self._signatures = ModelSignature(inputs=inputs, outputs=outputs, params=params)
```

Comment:
there is a specific method to get ModelSignature. Could you please move it there